pages/frontend/cuiloa.mdx

```diff
@@ -1,7 +1,91 @@
-import { Callout, Steps } from 'nextra/components'
-
 # Cuiloa
 
-<Callout emoji="💡">
-This section is incomplete. You can help by expanding it!
-</Callout>
+[Cuiloa] is a block explorer for navigating and inspecting data on [Penumbra]. While Penumbra is a fully private
+proof-of-stake network, Cuiloa allows you to explore the *public* facets of the Penumbra chain. Beyond just blocks and
+transactions, Cuiloa lets you explore staking, governance, and DEX activity along with Penumbra's cross-chain
+interactions via the Inter-Blockchain Communication (IBC) interoperability protocol, including established IBC clients
+running on Penumbra and recent IBC packets.
+
+## Using Cuiloa
```

Comment:
Can probably just delete this section for now if empty, we can revisit when something public exists